### PR TITLE
Added create player form html

### PR DESCRIPTION
**Pull request by thirdknife**
image

### CODE REVIEW

File: views/pages/players.html [hunk 1]
```diff
@@ -12,4 +12,95 @@
...
+        <div class="flex items-center justify-between">
+          <label for="name" class="block text-sm font-medium leading-6 text-gray-900">Full Name:</label>
+          <div class="mt-2">
+            <input type="text" id="name" name="name" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="birthdate" class="block text-sm font-medium leading-6 text-gray-900">Birthdate (yyyy-mm-dd):</label>
```
Comment (Omustardo) [inline views/pages/players.html]: From the screenshot, this says `yyyy-mm-dd` but the input field is `mm/dd/yyyy`

Comment (Omustardo) [reply views/pages/players.html]: We should probably use whatever users would be most familiar with, even if we store it as yyyy-mm-dd in the database.

File: views/pages/players.html [hunk 2]
```diff
@@ -12,4 +12,95 @@
...
+        <div class="flex items-center justify-between">
+          <label for="height" class="block text-sm font-medium leading-6 text-gray-900">Height (cm):</label>
+          <div class="text-sm">
+            <input type="number" id="height" name="height" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="weight" class="block text-sm font-medium leading-6 text-gray-900">Weight (grams):</label>
```
Comment (Omustardo) [inline views/pages/players.html]: Like with the birthdate above, perhaps kilos would be a better human readable input? We can convert to grams internally for the database, or just change the comment on the `weight` field in `database.Player` to say kgs.

File: views/pages/players.html [hunk 3]
```diff
@@ -12,4 +12,95 @@
...
+          <div class="text-sm">
+            <input type="text" id="manager_name" name="manager_name" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="telephone" class="block text-sm font-medium leading-6 text-gray-900">Telephone:</label>
+          <div class="text-sm">
+            <input type="tel" id="telephone" name="telephone" pattern="[0-9]{10}" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
```
Comment (Omustardo) [inline views/pages/players.html]: I'm not convinced that it's worth sanitizing telephone numbers with `[0-9]{10}`. That would have issues with country codes, and many commonly used formats like `+1 (555) 555-5555`

We probably won't be doing anything with this field except displaying it, so as long as we have some sort of length limit in the backend (which we'd want for many fields), I'd recommend leaving it as an unsanitized string.

File: views/pages/players.html (hunk 2 above)
Comment (Omustardo) [reply views/pages/players.html]: Are lbs or kgs more commonly used? I think we should stick with whichever is most widely used and not have to worry about conversions or rounding.

File: views/pages/players.html (hunk 1 above)
Comment (thirdknife) [reply views/pages/players.html]: I couldn't change the default input format, that stays as `mm/dd/yyyy` and there is no other way beside using js library so lets stick with default

File: views/pages/players.html (hunk 2 above)
Comment (thirdknife) [reply views/pages/players.html]: switched to lbs

**Comment by thirdknife** [comment]
Removed duplicate

**Pull request merged**